packages/editor/src/components/rich-text/index.native.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * External dependencies
+ */
+import RCTAztecView from 'react-native-aztec';
+import { children } from '@wordpress/blocks';
+
+/**
+ * WordPress dependencies
+ */
+import { Component, RawHTML, renderToString } from '@wordpress/element';
+import { withInstanceId, compose } from '@wordpress/compose';
+
+export class RichText extends Component {
+	constructor() {
+		super( ...arguments );
+		this.onChange = this.onChange.bind( this );
+		this.onContentSizeChange = this.onContentSizeChange.bind( this );
+
+		this.lastContentSizeHeight = -1;
```

Comment:
For the next PR. I don't think it is reliable to use private variables inside component to keep state. You should be using `this.setState` method instead which React provides by default: https://reactjs.org/docs/faq-state.html.

Reply:
Yeah, I used those private variables, instead keeping them in state, since even if the component gets recreated, and those variables reset to their initial value, the code works ok anyway.  Those variables control if `render` should be executed or not.